cupy/cuda/memory.pyx

```diff
@@ -470,8 +470,8 @@ cdef class MemoryPointer:
                 The default uses CUDA stream of the current context.
 
         """
+        ptr = mem if isinstance(mem, int) else mem.value
```

Comment:
I just checked. Could you move this line to the `if size > 0:` block below? This would align with the other function `copy_to_host_async` in the same file.

Reply:
No problem!
I will work on this tonight :)

Reply:
lets delete it in +473, now is dupped :)

Reply:
Ooops 😆